js/members.js

```diff
@@ -67,9 +76,52 @@ $("#batchButtons button").on("click", function() {
     $(this).addClass("active-batch");
 });
 
+// funtion to highlight selected tab on "allMembersButtons" Bar
+$("#allMembersButtons button").on("click", function() {
+    var buttons = $('#allMembersButtons').children();
+    for (button of buttons) {
+        $(button).removeClass("active-batch");
+    }
+    $(this).addClass("active-batch");
+   
+    // if "presentMembers" tab selected, make "all" as selected Tab on "batchButtons"
+    if(this == buttons[0])
```

Comment:
space after ```if```. I know it's minor but lets keep the code consistent.

Reply:
done